extension_config.cmake

```diff
@@ -15,4 +15,5 @@ duckdb_extension_load(httpfs
 	SOURCE_DIR ${CMAKE_CURRENT_LIST_DIR}
 	INCLUDE_DIR ${CMAKE_CURRENT_LIST_DIR}/extension/httpfs/include
 	${LOAD_HTTPFS_TESTS}
+	LINKED_LIBS "../../third_party/mbedtls/libduckdb_mbedtls.a"
```

Comment:
Is this necessary here?

Reply:
Unfortunately, yes, I have not found a clean way to clean this up.

It should picked up from CMake, but that requires some changes in duckdb/duckdb that are not there (and unsure if they can land today).

Reply:
I can fix this up, question is: it's a blocker?

Reply:
I don't know what problems it causes but it seems hacky - maybe it's fine. I'll leave it up to you